### PR TITLE
Reproducing experiment results for MS-MARCO passage and document ranking

### PR DESCRIPTION
**Pull request by pratyushpal**
Environment Details: 
+ MacBook Pro Apple M2, 16GB RAM, Ventura 13.3 
+ Apache Maven 3.9.2
+ Java version: 20.0.1
+ Python 3.10.9

### CODE REVIEW

File: docs/experiments-msmarco-doc.md [hunk 1]
```diff
@@ -289,3 +289,4 @@ That's it!
 + Results reproduced by [@Jocn2020](https://github.com/Jocn2020) on 2023-04-30 (commit [`30269d6`](https://github.com/castorini/anserini/commit/30269d636a556174e3756fe59accab6c22a7c732))
 + Results reproduced by [@zoehahaha](https://github.com/zoehahaha) on 2023-05-12 (commit [`b429218`](https://github.com/castorini/anserini/commit/b429218e52a385eabf3fd81979e221111fbc4a19))
 + Results reproduced by [@billcui57](https://github.com/billcui57) on 2023-05-14 (commit [`d82b6f7`](https://github.com/billcui57/anserini/commit/d82b6f76cbef009bccb0e67fc2bedc6fa6ae56c6))
++ Results reproduced by [@pratyushpal](https://github.com/pratyushpal) on 2023-07-14 (commit [`17d5fc7f`](https://github.com/castorini/anserini/commit/17d5fc7f338b511c4dc49de88e9b2ab7ea27f8aa))
```
Comment (lintool) [inline docs/experiments-msmarco-doc.md:292]:
```suggestion
+ Results reproduced by [@pratyushpal](https://github.com/pratyushpal) on 2023-07-14 (commit [`17d5fc7`](https://github.com/castorini/anserini/commit/17d5fc7f338b511c4dc49de88e9b2ab7ea27f8aa))
```

File: docs/experiments-msmarco-passage.md [hunk 2]
```diff
@@ -399,3 +399,4 @@ The BM25 run with default parameters `k1=0.9`, `b=0.4` roughly corresponds to th
 + Results reproduced by [@billcui57](https://github.com/billcui57) on 2023-05-11 (commit [`a913b5`](https://github.com/castorini/anserini/commit/a913b52478ae63e4427215e16f58081686f8e8f2))
 + Results reproduced by [@zoehahaha](https://github.com/zoehahaha) on 2023-05-12 (commit [`d82b6f7`](https://github.com/castorini/anserini/commit/d82b6f76cbef009bccb0e67fc2bedc6fa6ae56c6))
 + Results reproduced by [@Singularity-tian](https://github.com/Singularity-tian) on 2023-05-12 (commit ['d82b6f7'](https://github.com/castorini/anserini/commit/d82b6f76cbef009bccb0e67fc2bedc6fa6ae56c6))
++ Results reproduced by [@pratyushpal](https://github.com/pratyushpal) on 2023-07-14 (commit [`17d5fc7f`](https://github.com/castorini/anserini/commit/17d5fc7f338b511c4dc49de88e9b2ab7ea27f8aa))
```
Comment (lintool) [inline docs/experiments-msmarco-passage.md:402]:
```suggestion
+ Results reproduced by [@pratyushpal](https://github.com/pratyushpal) on 2023-07-14 (commit [`17d5fc7`](https://github.com/castorini/anserini/commit/17d5fc7f338b511c4dc49de88e9b2ab7ea27f8aa))
```

**Review by lintool — CHANGES_REQUESTED** [review]
We're doing 6-alphanum hash ids instead of 7?

File: docs/experiments-msmarco-passage.md (hunk 2 above)
Comment (pratyushpal) [reply docs/experiments-msmarco-passage.md]: Noted! Updated it with the requested changes.

**Review by lintool — APPROVED** [review]

**Pull request merged**